### ISSUE TITLE
Are you interested to help maintain the installr R Package? Please leave a comment here!

### ISSUE THREAD

**Issue by talgalili** [talgalili/installr#183]
Hello,

I am the current maintainer of the installr R package, which provides functions for the installation and updating of software on Windows OS, with a particular focus on R. Due to a shift away from Windows OS, I am seeking someone enthusiastic and dedicated to take over maintenance of the package.

If you are an experienced R developer with a strong background in Windows OS, and you are passionate about helping fellow R users, this might be a great opportunity for you to make a significant contribution to the community.

If you are interested in maintaining the installr package, please express your interest in this GitHub issue. Kindly provide details about your experience with R, Windows OS, and any other relevant experience you may have. I will review all expressions of interest and follow up with those that match the required skills and experience.

Thank you for your interest in maintaining the installr package and for your continued support of the R community.

( also posted here: https://www.r-statistics.com/2023/04/seeking-a-new-maintainer-for-the-popular-r-package-installr/ )

**Comment by StatsGary** [comment]
Yeah happy to own it.

**Comment by talgalili** [comment]
Wonderful.
Could you please take a look at open issues and see if you have any
insights/questions?

**Comment by talgalili** [comment]
Hey @StatsGary - I've sent you an invite to the repo. Could you please take a look?
Also, would you be available to go over the open issues and seeing what can be done there?

**Comment by BroVic** [comment]
Hi @talgalili , are you looking for co-maintainers or for someone to take over? I could go with the former, not the latter.

**Comment by talgalili** [comment]
Hey,
Co maintainer would be great :)

**Comment by BroVic** [comment]
Okay, I'm currently on life's "hot seat", but I will try to find time to address issue #184 for starters. If that's alright with you, will do a PR when done.

**Comment by talgalili** [comment]
Sounds great!

**Comment by StatsGary** [comment]
Sure. Found this in my junk, so apologies.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by talgalili** [comment]
No worries, thanks.

**Comment by burgerga** [comment]
To be fair, I have used and contributed to installr in the past. But I now completely moved to [winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/) (+[wingetui](https://www.marticliment.com/wingetui/)). 
It's not as polished yet as linux package managers but it's getting closer, and it's really nice that I can now install/update everything from one location:
![image](https://github.com/talgalili/installr/assets/907362/19d7f667-09d0-4aae-912b-67f6a679d291)
So, it would not be unreasonable IMO to sunset installr 🤷

**Comment by BroVic** [comment]
Can winget manage local R packages (i.e. post-installation) the way `{installr}` can?

**Comment by burgerga** [comment]
@BroVic No, but `installr` also doesn't handle that well I would say (see e.g. https://github.com/talgalili/installr/issues/172).

So typically I do 

```r
update.packages(ask = F, checkBuilt = T)
```

or I start completely clean and install everything I need via [pak](https://github.com/r-lib/pak) (I typically have all my dependencies in a DESCRIPTION file, in my first code chunk, or in [renv](https://rstudio.github.io/renv/articles/renv.html)).

**Comment by lichenlady94** [comment]
![1](https://github.com/talgalili/installr/assets/108133706/677ce569-5148-4b83-9819-559ff4ab3e3a)

Hi, I tried to do install.packages("installr") and then library(installr) but when I was at the library command, I got the message that's in the attached picture. I was wondering if there's anything going on atm that's making it hard to do the update.

**Comment by burgerga** [comment]
@lichenlady94 No this is the expected behavior, the package should still work :)

**Comment by kjngo0309** [comment]
Hi, I was running updateR() is it suppose to provide some prompt about what the latest version of R is and wait for us to note how to proceed? Currently it was just hanging there and not providing anything. Thanks!

**Comment by burgerga** [comment]
@kjngo0309 Can you make this into a separate issue, then we can keep discussion on this issue clean

**Comment by kjngo0309** [comment]
> @kjngo0309 Can you make this into a separate issue, then we can keep discussion on this issue clean

Issue resolved. Thanks

**Comment by burgerga** [comment]
It's been a year now, and another year I (as a collaborator to the installr package) haven't even used the package (see https://github.com/talgalili/installr/issues/183#issuecomment-1725363286). I do however get all issues in my mailbox, so what I propose to do is the following:

* Add a badge with repostatus unsupported (see https://www.repostatus.org/#unsupported for definition) to the repo (the pull request linked above).
* Since so many people still use it, we should push one final update to CRAN where we inform the users that installr is not being maintained, and that alternatives are available (and potentially remove the update packages function (#172) to avoid issues for those users)
* Then, finally, thank @talgalili for all his work on the package 👏  it was really useful and filled a need :)

**Comment by lichenlady94** [comment]
Thanks! If you need additional help, please let me know, I'm getting into the space and enjoy learning more!

**Comment by DavisVaughan** [comment]
@burgerga Rig is also a very actively maintained way to manage R installations on Mac, Windows, and Linux that most of us have been using for a few years now: https://github.com/r-lib/rig. It seems like it might also be reasonable to link people over to rig from the installr readme if you really are considering sunsetting it.

In general rig tries to "do the right thing" with R installations to avoid as many issues as possible that we've seen over the years (like managing the Windows registry, helping install Rtools, requiring a user level package library on Mac, etc). https://github.com/r-lib/rig?tab=readme-ov-file#-features-

**Comment by burgerga** [comment]
@DavisVaughan Ah I did not know that one yet, very useful!
I think the combination of rig (for R & Rtools) and winget for everything else (including installing rig ;) ) would cover the use cases of installr, but ultimately the decision of sunsetting installr is up to @talgalili as he is the original author & CRAN maintainer
